packages/styles/forms.css

```diff
@@ -16,6 +16,7 @@
   --field-label-text-color: var(--gray-90);
   --field-label-description-text-color: var(--gray-60);
   --input-min-width: 250px;
+  --focus-glow: rgba(181, 26, 209, 1);
```

Comment:
We want to pull from global vars here:
https://github.com/dequelabs/cauldron/blob/720bc9bd5db3eab975d0c6d80639c8614f3899a4/packages/styles/variables.css#L41-L42

We also already have `focus-glow` defined in vars so we probably need to rescope the var here. Ideally following our guidelines it should be descriptive something like:

```css
--field-focus-ring-color: var(--focus-light);
```

With the alternative set for dark mode as well.